configurable_lti_consumer/static/js/configurable_xblock_lti_consumer.js

```diff
@@ -1,9 +1,14 @@
-function configurableLTIConsumerXblockIframeResizerInit() {
-    $(function ($) {
-        LtiConsumerXBlock();  // call inherited lti_consumer javascript initialization
-        $('.ltiLaunchFrame').iFrameResize({ // Initialize IframeResizer on lti_consumer iframe class
-            checkOrigin: false,
-            log: false
-        })
-    });
+function configurableLTIConsumerXblockIframeResizerInit(runtime, element, json_args) {
+    var $element = $("#" + json_args["element_id"]);
+    LtiConsumerXBlock();  // call inherited lti_consumer javascript initialization
+    // Estimate LTI consumer xblock height regarding available width using given ratio
+    // to presize iframe before iframeResizer initialization
+    var ratio = parseFloat($element.find("div").data('ratio'));
```

Comment:
what happens if I don't configure a ratio. Maybe it should default to the `inline_height` setting WDYT?

Reply:
default value in Python code:
```
configuration.get("automatic_resizing_initialization_ratio", 0.5625)
```